plugins/lime-plugin-locate/src/communityLayer.js

```diff
@@ -0,0 +1,75 @@
+const isObject = (obj) => typeof obj === 'object' && obj !== null;
+
+const sameLocation = (coord1, coord2) => Number(coord1.lon) === Number(coord2.lon) && Number(coord1.lat) === Number(coord2.lat);
+
+const removeDuplicates = (collection, getIdentifier) => {
+    const identifierState = {};
+    return collection.filter((value) => {
+        const identifier = String(getIdentifier(value));
+        if (identifierState[identifier]) {
+            return false;
+        }
+        identifierState[identifier] = true;
+        return true;
+    });
+};
+
+const coordsToPoint = ({ coordinates, hostname }) => ({ type: 'Feature', geometry: { type: 'Point', coordinates: [Number(coordinates.lon), Number(coordinates.lat)] }, properties: { name: hostname } });
```

Comment:
I would rename hostname to `additiona-props`, and add it wherever this is called from, as it breaks the meaning of the function, because it has a side effect and because it crosses boundaries, from geo to network.

Reply:
I have made the change as you suggested! :)